openmeetings-web/pom.xml

```diff
@@ -494,6 +494,10 @@
 			<groupId>org.apache.openmeetings</groupId>
 			<artifactId>openmeetings-core</artifactId>
```

Comment:
`-util, -db and -core` are redundant

Reply:
I did not modify those.

Reply:
why are they redundant? They are required to be packaged up in the .war file. How would they be linked into the war file of the -web project if they are not defined as dependencies?